### ISSUE TITLE
NVDA fails to read the content of the HTML editor of Google Blogger

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#13538]
### Steps to reproduce:
1. Prepare a simple text with a few lines of text, for the testing purposes.
2. Open www.blogger.com in your web browser and sign in with your account.
3. Activate the "New post" command from the page.
4. Above the text area for the post there is a list box control to switch between the HTML mode and the other (WYSIWYG) mode for the editor - select the HTML mode.
5. Copy the text from step 1 and paste it into the post editor of Blogger.
6. Use the arrow keys to read the pasted text.
7. From the control from step 4, select the WYSIWYG mode for the editor.
8. Repeat steps 5 and 6.

### Actual behavior:
In step 6, NVDA does not read anything.

### Expected behavior:
NVDA should read the pasted text content, as it does in step 8.

### System configuration
- Operating system: Windows 11 Pro version 21H2 (build 22000.556), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: 2021.3.5 portable, alpha-25076,14a94998 installed, both in bulgarian.
- Web browser version: Microsoft Edge 99.0.1150.52, 64-bit, in bulgarian; Mozilla Firefox 98.0.2, 64-bit, in bulgarian.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your computer?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Both 2021.3.5 and alpha-25076,14a94998 - the behavior is the same.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, it is.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I haven't tried that one yet, but considering the overall behavior of NVDA, it is highly unlikely that it will solve the problem.

**Comment by feerrenrut** [comment]
This seems to be a problem with the blogger platform. Please report this feedback to their support channel.
There is nothing we can do about this, closing this issue as needs external fix.
